automl/api/Snippets/BatchPredict.cs

```diff
@@ -0,0 +1,105 @@
+// Copyright (c) 2019 Google LLC.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not
+// use this file except in compliance with the License. You may obtain a copy of
+// the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+// WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+// License for the specific language governing permissions and limitations under
+// the License.
+
+using CommandLine;
+using Google.Cloud.AutoML.V1;
+using System;
+
+namespace GoogleCloudSamples
+{
+    [Verb("batch_predict", HelpText = "Batch Predict your files and save the results to GCS")]
+    public class BatchPredictOptions : PredictOptions
+    {
+        [Value(2, HelpText = "GCS location of your csv or jsonl file.")]
+        public string InputUri { get; set; }
+
+        [Value(3, HelpText = "GCS location to save the results.")]
+        public string OutputUri { get; set; }
+    }
+    class AutoMLBatchPredict
+    {
+        // [START automl_batch_predict]
+        /// <summary>
+        /// Demonstrates using the AutoML client to predict image contents.
+        /// </summary>
+        /// <param name="projectId">GCP Project ID.</param>
+        /// <param name="modelId">the Id of the model.</param>
+        /// <param name="inputUri">The GCS path where all images are.</param>
+        /// <param name="outputUri">The GCS path to store the output of your prediction request.</param>
+        public static object BatchPredict(string projectId = "YOUR-PROJECT-ID",
+            string modelId = "YOUR-MODEL-ID",
+            String inputUri = "gs://YOUR_BUCKET_ID/path_to_your_input_csv_or_jsonl",
```

Comment:
If I add images or text contents, it will be lengthy and since we already mentioned above twice, can I just leave as it is for this param? or Should I add comment saying --> //images or text contents right next to it?